src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.ts

```diff
@@ -11,17 +11,16 @@ import { MicronutrientMeasureType } from 'src/app/apiAndObjects/objects/enums/mi
   styleUrls: ['./quickMapsHeader.component.scss'],
 })
 export class QuickMapsHeaderComponent {
-  public ROUTES = AppRoutes;
+  public readonly ROUTES = AppRoutes;
+  public readonly MEASURE_TYPES = MicronutrientMeasureType;
+
   constructor(
     public route: ActivatedRoute,
     public quickMapsService: QuickMapsService,
     private sharingService: SharingService,
-  ) {}
+  ) { }
```

Comment:
what is causing this space? prettier rules do not add this so must be something local to your vscode setup?

Reply:
I'll have a look what I've got enabled.  Probably a vscode plugin...

Reply:
VSCode output tab will let you see which extension is doing the modify on save
![image](https://user-images.githubusercontent.com/4210022/116385662-ea5f7680-a810-11eb-9c4a-fa601003fb15.png)